backend/src/services/services.ts

```diff
@@ -82,6 +83,19 @@ const http = new Http();
 const udemy = new Udemy({
   httpService: http,
   baseUrl: ENV.UDEMY.BASE_URL,
+  clientData: {
+    clientId: ENV.UDEMY.CLIENT_ID,
+    clientSecret: ENV.UDEMY.CLIENT_SECRET,
+  },
```

Comment:
```suggestion
    clientId: ENV.UDEMY.CLIENT_ID,
    clientSecret: ENV.UDEMY.CLIENT_SECRET,
```
let's avoid nesting. don't forget to remove type